internal/cli/streams/processor/delete_test.go

```diff
@@ -0,0 +1,33 @@
+package processor
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/mocks"
+)
+
+func TestDeleteOpts_Run(t *testing.T) {
```

Comment:
There should be a `t.Run()` call in this test describing what is being tested/asserted.